tests/acceptance/test_streaming_data_measurement.py

```diff
@@ -0,0 +1,143 @@
+"""Tests to validate a v2 measurement service that streams data."""
+from typing import Generator, List
+
+import grpc
+import pytest
+from examples.sample_streaming_measurement import measurement
+from google.protobuf import any_pb2
+
+from ni_measurementlink_service._internal.stubs.ni.measurementlink.measurement.v2 import (
+    measurement_service_pb2 as v2_measurement_service_pb2,
+    measurement_service_pb2_grpc as v2_measurement_service_pb2_grpc,
+)
+from ni_measurementlink_service.measurement.service import MeasurementService
+from tests.assets import sample_streaming_measurement_test_pb2
+
+
+@pytest.mark.parametrize("num_responses", [1, 10, 100])
+def test___streaming_measurement_service___request_number_of_responses___receives_responses(
+    num_responses: int, stub_v2: v2_measurement_service_pb2_grpc.MeasurementServiceStub
+):
+    request = v2_measurement_service_pb2.MeasureRequest(
+        configuration_parameters=_get_configuration_parameters(num_responses=num_responses)
+    )
+
+    response_iterator = stub_v2.Measure(request)
+    responses = [response for response in response_iterator]
+    assert len(responses) == num_responses
```

Comment:
3 sections: arrange, act, assert
```suggestion
    response_iterator = stub_v2.Measure(request)

    responses = [response for response in response_iterator]
    assert len(responses) == num_responses
```